packages/itwin/tree-widget/src/tree-widget-react.ts

```diff
@@ -3,8 +3,8 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-export { TreeWidget } from "./TreeWidget.js";
-export { createTreeWidget, TreeWidgetComponent } from "./components/TreeWidgetUiItemsProvider.js";
-export { SelectableTree, SelectableTreeDefinition } from "./components/SelectableTree.js";
-export { TreeWithHeader } from "./components/tree-header/TreeWithHeader.js";
-export * from "./components/trees/index.js";
+export { TreeWidget } from "./tree-widget/TreeWidget.js";
```

Comment:
Maybe source code folder should be renamed to `tree-widget-react` to match barrel file and package name. Same as you did for property grid?

Reply:
agree, changed